ui_components/src/home/News.js

```diff
@@ -230,4 +234,11 @@ function NewsButtons({ loadNews, createPost, isAdmin }) {
             }
         </div>
     );
+}
+
+// Should probably be a global helper
+function clickOnEnter(event) {
+    if (event.key === "Enter") {
+        event.target.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+    }
```

Comment:
This is a global helper now, we can remove from here

Reply:
Resolved in commit.